Monika After Story/game/script-holidays.rpy

```diff
@@ -7264,7 +7264,12 @@ image chibi_peek = MASFilterSwitch("mod_assets/other/chibi_peek.png")
 label mas_bday_surprise_party_hint:
     #Set up letters
     python:
-        persistent._mas_bday_hint_filename = "For " + player + ".txt"
+        player_sane = player
+        if renpy.windows:
```

Comment:
just remove the extra stuff for all os, linux will get annoyed with some of those too

Reply:
Got it.